app-modules/form/routes/web.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+use App\Livewire\EmbedForm;
+use Assist\Form\Models\Form;
+use Illuminate\Http\Request;
+
+Route::middleware('web')
+    ->prefix('forms/embed')
+    ->name('forms.embed.')
+    ->group(function () {
+        // Route::get('/{form}', function (Form $form) {
+        //     dd($form->loadMissing('items'));
+        // })->name('show');
+
+        Route::get('/{embed}', EmbedForm::class)
```

Comment:
We could change `embed` for `form` here to make clearer what should be passed.

We may want to change `EmbedForm` to something like `RenderedForm` but not necessary.

Reply:
I went through and changed it. I didn't do this before because it conflicts with the default behavior of Filament. I got around this, but it might make it slightly more cumbersome to work with.